rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -2780,6 +2792,7 @@ extern "C" rmw_ret_t rmw_publisher_event_init(
 extern "C" rmw_ret_t rmw_subscription_event_init(
   rmw_event_t * rmw_event, const rmw_subscription_t * subscription, rmw_event_type_t event_type)
 {
+  RET_NULL(subscription);
   RET_WRONG_IMPLID_X(subscription, return RMW_RET_INCORRECT_RMW_IMPLEMENTATION);
```

Comment:
can be replaced with RET_WRONG_IMPLID?

Reply:
Indeed. See  https://github.com/ros2/rmw_cyclonedds/pull/226/commits/4432c12c86be2d44ad7f57249339d2756bdbe98d.